tests/js/babel-plugins/module-blocks-range.js

```diff
@@ -0,0 +1,4 @@
+let moduleBlock = module {  <<<PRETTIER_RANGE_START>>>export let y = 1;
```

Comment:
Let's move this file to `tests/js/module-blocks`

Reply:
What about `babel-plugins/module-blocks.js`? Is it intentionally there?

Reply:
This dir was  added to test syntax only https://github.com/prettier/prettier/blob/2a253828b79c07a8ce5c64cc09208d89df37701c/src/language-js/parser-babel.js#L24